.eslint-dictionary.json

```diff
@@ -371,6 +383,7 @@
   "utils",
   "validators",
   "venv",
+  "verificationbucket",
```

Comment:
nit: can this be replaced in code with `verificationBucket`

Reply:
This cannot be safely changed as far as I can tell.